packages/react-ui-core/src/Counter/Counter.js

```diff
@@ -66,6 +67,7 @@ export default class Counter extends Component {
             )}
           >{`${decrementUnit} ${count} ${incrementUnit}`}</span>
           <span
+            id="decrement"
```

Comment:
`id`s should _never_ be in these components. If you want to pass an `id` that's fine, but it should be spread out on the top level component `{...props}` (this is the rest of `this.props`, not all of them.